lucien-ios/ProfileViewController.swift

```diff
@@ -16,18 +16,56 @@ final class ProfileViewController: UIViewController {
     @IBOutlet private weak var name: UILabel!
```

Comment:
These labels should have `label` appended to their names, otherwise it looks like they're data and not UI elements.